bindings/profilers/cpu.cc

```diff
@@ -0,0 +1,268 @@
+#include <queue>
+
+#include <v8-profiler.h>
+#include <v8.h>
+#include <node.h>
+#include <uv.h>
+
+#include "../per-isolate-data.hh"
+#include "../location.hh"
+#include "cpu.hh"
+
+namespace dd {
+
+CpuProfiler::CpuProfiler()
+  : isolate_(v8::Isolate::GetCurrent()),
+    async(new uv_async_t()),
+    code_map(CodeMap::For(isolate_)),
```

Comment:
Possibly a redundant question: 
Can we assume the code_map pointer is stable ?

Reply:
Yep. The CodeEventHandler class is not GC-managed.